.github/workflows/ci.yaml

```diff
@@ -89,15 +89,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.3, 7.4]
-        sylius: [1.8.*, 1.9.*]
-        symfony: [^4.4, ^5.2]
+        php: [7.3, 7.4, 8.0]
+        sylius: [1.8.0, 1.9.0, 1.10.0]
```

Comment:
Sylius Stantard 1.8 install is KO, so i remove it from the CI, but Plugin work fine with this version.